File: pgm/ABC_Visual_Basic/ABC_165_B.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim X As Long = Console.ReadLine()
        Dim cur As Long = 100
        Dim ans As Integer = 0
        While cur < X
            cur += Math.Floor(CType(cur, Decimal) / 100)
            ans += 1
        End While
        Console.WriteLine(ans)
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_165_B.cs
using System;
// using System.Collections.Generic;
// using System.Linq;

class Program
{
    static void Main()
    {
        long X = long.Parse(Console.ReadLine());
        long cur = 100L;
        long ans = 0L;
        while (cur < X)
        {
            cur += cur / 100;
            ans++;
        }
        Console.WriteLine(ans);
        return;
    }
}